RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/RobotMap.java

```diff
@@ -47,7 +47,6 @@
     public static DigitalInput limitSwitchBottom; // May be used for elevator as a sensor for testing if at the bottom of elevator
     public static DigitalInput limitSwitchTop; // May be used for elevator as a sensor for testing if at the top of elevator
     // Potentiometer
-    private static int inputPort = 1; // TODO find correct port for the potentiometer
 	private static int scale = 1; // TODO find correct scale for the potentiometer

```

Comment:
I think you've got this indented too far; also, assuming that's a constant you should make it `final` and UPPERCASE